Documentation/build-container-examples/postgres/postgres-prestart.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# Adapted from
+# https://github.com/docker-library/postgres/blob/b84ddd1/10/docker-entrypoint.sh
```

Comment:
should you also say that it's under MIT license?

Reply:
you can also say that the difference is that you don't do the `exec "$@"` at the end because this script will be executed from the pre-start hook (see `acbuild --debug set-event-handler pre-start` in `build-postgres.sh`)

Reply:
ok